streaming-bytestring.cabal

```diff
@@ -232,7 +232,7 @@ test-suite test
     , transformers
     , tasty >= 0.11.0.4
     , tasty-smallcheck >= 0.8.1
-    , tasty-hunit >= 0.10
+    , tasty-hunit >= 0.9
```

Comment:
I added this dependency in a previous PR, and it looks like `0.10` was too aggressive of a lower bound.